### DISCUSSION TITLE
.execute not working

### DISCUSSION THREAD

**Discussion by AlexInCube** [sidorares/node-mysql2#1600] [Q&A]
mySQLconnection.execute(
    'UPDATE slot_stats SET total_games = total_games+1, total_wins = total_wins + ?, jackpots = jackpots + ? WHERE user_id = ?',
    [win, jackpot, userId],
    function (err) {
      if (err) throw err
    })

This query works fine when I use .query . Now I want to use prepared statement and encounter the problem that .execute does not execute the query and I get no error either.

win, jackpot is 0 or 1, userID is a user id from discord

**Comment by AlexInCube** [comment]
```js
mySQLconnection.execute(
    'UPDATE slot_stats SET total_games = total_games + 1 WHERE user_id = ?;' +
    'UPDATE slot_stats SET total_wins = total_wins + ? WHERE user_id = ?;' +
    'UPDATE slot_stats SET jackpots = jackpots + ? WHERE user_id = ?;',
    [win, jackpot, userId],
    function (err) {
      if (err) throw err
    })
```

I split one big query into 3 small and setting multipleStatements: true and I get 
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the correct syntax to use near 'UPDATE slot_stats SET total_wins = total_wins + ? WHERE user_id = ?;UPDATE slot_' at line 1

**Comment by AlexInCube** [comment]
when i make 'UPDATE slot_stats SET total_games = total_games + 1 WHERE user_id = ?;'
execute is working, but i dont want to do multiple queries, it's not convenient

**Comment by sidorares** [comment]
> This query works fine when I use .query . Now I want to use prepared statement and encounter the problem that .execute does not execute the query and I get no error either.

what do you get in the response? do you see `affectedRows: 0`?

**Reply by AlexInCube** [reply]
Yes

**Reply by sidorares** [reply]
this means the execute call was successful but `WHERE user_id  = ?` didn't match anything. Double check the query and parameter

**Reply by AlexInCube** [reply]
if I write one of these queries, they work, but when I start listing columns, the query stops working.

'UPDATE slot_stats SET total_games = total_games + 1 WHERE user_id = ?;'
 'UPDATE slot_stats SET total_wins = total_wins + ? WHERE user_id = ?;'
 'UPDATE slot_stats SET jackpots = jackpots + ? WHERE user_id = ?;'
